python_modules/dagster-graphql/dagster_graphql/schema/pipelines/pipeline.py

```diff
@@ -592,6 +594,24 @@ def resolve_presets(self, _graphene_info):
     def resolve_isJob(self, _graphene_info):
         return self._external_pipeline.is_job
 
+    def resolve_isAssetJob(self, graphene_info):
+        handle = self._external_pipeline.repository_handle
+        location = graphene_info.context.get_repository_location(handle.location_name)
+        repository = location.get_repository(handle.repository_name)
+        return bool(repository.get_external_asset_nodes(self._external_pipeline.name))
```

Comment:
Curious if there's any overhead to get_external_asset_nodes - if we need to, we could just ask for assetNodes and keep your "if >0 assetNodes, it's an asset job" rule client-side.

Reply:
yeah, there shouldn't be any overhead, since we're storing them by job at the external repository level anyway.